src/app/components/ParaTimePicker/LayerDetails.tsx

```diff
@@ -80,6 +81,7 @@ const contentMinHeight = '270px'
 export const LayerDetails: FC<LayerDetailsProps> = ({ network, selectedLayer }) => {
   const { t } = useTranslation()
   const theme = useTheme()
+  const isTablet = useMediaQuery(theme.breakpoints.down('md'))
```

Comment:
Can't we use the newly available `useScreenSize()` for this? It already provides `isTable`. I  think it would be confusing to use different definitions at multiple points of the code... right?

Reply:
Right, leftover from the rebase. Nice catch 👍